contrib_bots/lib/DefineBot/docs.md

```diff
@@ -0,0 +1,21 @@
+# DefineBot
+
+This is a bot that defines a word that the user inputs. Whenever the user
+inputs a message starting with '@define', the bot defines the word
+that follows.
+
+&nbsp;&nbsp;&nbsp;&nbsp;The definitions are brought to the website using an API. The bot posts the
```

Comment:
Yeah, but that's abusing Markdown's inline HTML functionality. If you really want indentation you could use bullet points instead.